go.mod

```diff
@@ -3,22 +3,21 @@ module github.com/metal3-io/ip-address-manager
 go 1.16
 
 require (
-	cloud.google.com/go v0.75.0 // indirect
```

Comment:
why is this getting removed?

Reply:
Most probably because of the `go mod tidy`.